9c-internal/multiplanetary/network/heimdall.yaml

```diff
@@ -341,12 +341,16 @@ stateMigrationService:
 
 arenaService:
   enabled: true
-  rwMode: false
 
   nodeSelector:
     read:
       eks.amazonaws.com/nodegroup: heimdall-spot_2c
 
-  env:
-  - name: Worker
-    value: "true"
+  graphqlEndpoint: "https://heimdall-internal-test-1.nine-chronicles.com/graphql"
```

Comment:
`https://heimdall-internal-rpc-1.nine-chronicles.com/graphql`??
no test headless for internal networks.
or internal domain would be better `http://remote-headless-1.heimdall.svc.cluster.local/graphql`